.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "sdsl-lite"]
 	path = submodules/sdsl-lite
-	url = https://github.com/xxsds/sdsl-lite
+	url = https://github.com/eseiler/sdsl-lite
```

Comment:
This was not supposed to be committed, right?

Reply:
Nope, I just wanted to check if CI runs through with my SDSL PR